base/uk.ac.stfc.isis.ibex.validators/src/uk/ac/stfc/isis/ibex/validators/PvValidator.java

```diff
@@ -56,7 +56,7 @@ public Boolean validatePvAddress(String pvAddress) {
 
         if (pvAddress.isEmpty()) {
             setErrorMessage(ADDRESS_EMPTY);
-        } else if (!(pvAddress.matches("^[a-zA-Z0-9_:]+(\\.[a-zA-Z0-9_:]+)?$"))) {
+        } else if (!(pvAddress.matches("^[a-zA-Z0-9_\\-:]+(\\.[a-zA-Z0-9_:\\-]+)?$"))) {
```

Comment:
I noticed that this currently allows PVs with multiple colons (e.g. `IN:DEMO::SOME:PV`). Would it make sense to fix this as part of this PR?

Reply:
I had a quick look at this but it's not completely trivial, you need to craft a regex that allows any number of colon-seperated strings but disallows consecutive colons. Definitely possible but I'd rather not mix it in with this change.

I've created ticket https://github.com/ISISComputingGroup/IBEX/issues/2644 to take a look at this.